test/unit/data/model/migrations/test_dbscript.py

```diff
@@ -130,19 +131,8 @@ def dburl_from_config(config: Config) -> str:
 
 
 def run_command(cmd: str) -> subprocess.CompletedProcess:
-    if in_packages():
-        cmd = f"../.{cmd}"  # if this is run from `packages`, manage_db.sh is in parent directory
-
-    completed_process = subprocess.run(cmd.split(), capture_output=True, text=True)
-
-    return completed_process
-
-
-def in_packages() -> bool:
```

Comment:
I mean... if in packages skip the test is what I'd do here. The package tests should pretend they don't have galaxy_directory() available because they're meant to be installed into a Python env 🤔. This test is fundamentally not testing anything about the Galaxy data package - it is testing scripts placed in Galaxy root directory and that would not be available in an installed Galaxy environment. I would also move this test to... the app package instead of the data package for that reason I think.

Reply:
That never occurred to me.. It makes perfect sense, of course. Will make the changes, thanks!